src/solver/main.jl

```diff
@@ -0,0 +1,293 @@
+# This file uses Groebner, so it needs to be added in the future.
+
+"""
+    solve(expr, x, multiplicities=false)
```

Comment:
This does not match the documented CommonSolve.jl interface.

Reply:
Would the correct signature be `CommonSolve.solve(prob::Any; kwargs...)` ?
I think we would like to accept basically anything as `prob` (a constant, an expression, an array).

Alternatively, it can be `CommonSolve.solve(prob::SymbolicEquationsProblem; kwargs...)`. The user would construct `prob = SymbolicEquationsProblem(expr, x)` and pass it to `solve`, but that does not seem convenient.